### DISCUSSION TITLE
Willkommen zum Audio Developer Stammtisch Hamburg!

### DISCUSSION THREAD

**Discussion by scheffle** [scheffle/hamburg_audio_dev_stammtisch#1] [Announcements]
## 👋 Willkommen!

Hier können gerne Termine und Orte besprochen werden.

**Comment by thomaskvnze** [comment]
Hey!

wer organisiert denn die treffen? Ich bin am überlegen ein community treff in hamburg zu gestalten und bin gerade auf das github gestoßen

**Reply by scheffle** [reply]
Moin,
die Organisation soll hier über dieses Repository organisiert werden. Wir hatten letzte Woche erst den ersten Stammtisch. Ist also alles noch nicht sonderlich durch organisiert.

**Reply by thomaskvnze** [reply]
Okay, ich würde gerne vorbeikommen.

**Reply by scheffle** [reply]
Na klar, Du bist herzlichst Willkommen!
